src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -337,35 +332,6 @@ public string GenerateFile()
 			Console.WriteLine("Processing file {0}".InvariantCultureFormat(_fileDefinition.FilePath));
 #endif
 
-			// Check for the Roslyn generator's output location
-			var outputFile = Path.Combine(
-				_targetPath,
-				$@"g\{typeof(XamlCodeGenerator).Name}\{_fileDefinition.UniqueID}.g.cs"
-				);
```

Comment:
By default, the files are not written to disk. So, in most cases this code does nothing.

The code isn't correct anyway. The fact that XAML file didn't change doesn't mean that the generated code will remain the same.